pkg/mount/mountutil.go

```diff
@@ -19,11 +19,17 @@ package mount
 import (
 	"bufio"
 	"fmt"
+	"io"
 	"os"
 	"path/filepath"
 	"strings"
 )
 
+var ErrCouldNotFindRootDevice = fmt.Errorf("could not find root device")
+
+const ProcCmdLine = "/proc/cmdline"
+const HostProcCmdLine = "/host/" + ProcCmdLine
```

Comment:
```suggestion
const procCmdLine = "/proc/cmdline"
const hostProcCmdLine = "/host/" + procCmdLine
```
Lets keep both of them unexported for now. WDYT?

Also, should we change it to 
```"/host" + procCmdLine```, else the resulting path will be `/host//proc/cmdline`

Reply:
oh good catch

Reply:
done